src/components/TradeWidget/TxMessage.tsx

```diff
@@ -49,6 +124,15 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken })
         Your order starts {formatTimeInHours(validFrom || 0, 'now')}{' '}
         {validUntil && `and will expire ${formatTimeInHours(validUntil, 'Never')}`}
       </p>
+      {!isLoading && isLowVolume && (
+        <Warning>
+          <span>
+            This is a small order. Please keep in mind that solvers might not include your order if it does not generate
```

Comment:
Not sold on the wording. Maybe:

```suggestion
            This is a low volume order. Please keep in mind that solvers might not include your order if it does not generate
```